sdk/src/rpc/types/request.ts

```diff
@@ -8,6 +8,8 @@ import {
     RPCAppSessionAllocation,
     RPCAllowance,
     GetLedgerTransactionsFilters,
+    RPCAppStateIntent,
```

Comment:
On BE side it is called `AppSessionIntent`, I'm not sure whether it is better name than this, but let's have the same for TS and Golang

Reply:
Agh, okay..